server/base/src/main/java/org/apache/accumulo/server/conf/util/ZooInfoViewer.java

```diff
@@ -195,16 +191,17 @@ private void printProps(final InstanceId iid, final ZooReader zooReader, final O
     }
 
     if (opts.printTableProps()) {
-      Map<String,VersionedProperties> tProps = fetchTableProps(iid, opts.getTables(), zooReader);
+      Map<String,VersionedProperties> tProps = fetchTableProps(context, opts.getTables());
       writer.println("Tables: ");
       printSortedProps(writer, tProps);
     }
     writer.println();
   }
 
-  private void printIdMapping(InstanceId iid, ZooReader zooReader, PrintWriter writer) {
+  private void printIdMapping(ServerContext context, PrintWriter writer) {
+    var iid = context.getInstanceID();
```

Comment:
Could inline this, seems like its only used once.

Reply:
Yeah we could, but this follows the same pattern as the other methods, so it's fine for now. It may get inlined in a future change, though, after we chroot the zk objects, so we don't need to use instance id everywhere to retrieve from zk.